src/modules/admin/components/services/ServiceTypeDetailPage.tsx

```diff
@@ -88,15 +89,27 @@ const ServiceTypeDetailPage = () => {
       <Stack gap={2}>
         <Paper sx={{ p: 2 }}>
           <Stack gap={1}>
-            <CommonLabeledTextBlock title='Service Category'>
-              {serviceType.category}
-            </CommonLabeledTextBlock>
-            <CommonLabeledTextBlock title='Service Type'>
+            <CommonLabeledTextBlock title='Service Type Name'>
```

Comment:
Still todo (per ticket instructions): Build in some helpful instructions about how to use services/service types/forms

Reply:
Ok, I've added some pretty low-effort instructions here. Should we spend more time to make this more self-explanatory, possibly consult with design?

Reply:
I think this is nice! I added a few suggestions but I say lets go with this.